ionic.starter.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "JHipster Starter",
-  "baseref": "504dfeec2df39683cdb1500d34853582ca9b8df7",
+  "baseref": "master",
```

Comment:
This is a risky change and probably not something we want to do. It'll pin the versions that are used in the starter to the latest ones on the master branch of the Ionic Team's starter project. It's better to pin things to a specific commit so you know what versions of libraries to expect.

Reply:
Sure.

I checked the commits on the ionic-team starter project and it's mostly bumping versions so maybe I can update baseref to the latest commit.
It will also remove some security vulnerabilities in package.json dependencies